docs/3.0.0-beta.x/guides/databases.md

```diff
@@ -291,4 +296,16 @@ Replace the contents of `database.json` with the following:
 }
 ```
 
+::: warning Note
+The above configuration will create a database called `test`, the default database for **MongoDB Atlas**. If you would like to name your database something else, add the following **key:value pair** into your **"settings":** in your `database.json` file.
```

Comment:
This information is not entirely correct.
Strapi will not use the database name that is set in the `uri` variable (so it will not be `test`)
The default database name is set [here](https://github.com/strapi/strapi/blob/master/packages/strapi-hook-mongoose/lib/index.js#L45) and it will be `strapi`.

We have to update this note.

Reply:
This is now resolved as per your notes above.